scripts/summarize-report.js

```diff
@@ -296,7 +296,6 @@ if (!args['exclude-requirements-matrix']) {
   console.log('Requirement ID | Test Procedures | Status');
   console.log('--- | --- | --- ');
   _.each(sortedRequirements, function (requirement) {
-    var procedureCount = requirement.procedures.length;
```

Comment:
Do we even use this file anymore? should we just axe it?

Reply:
probably. can create a maintenance ticket to prune unnecessary files

Reply:
https://github.com/NASA-AMMOS/openmct-mcws/issues/285